<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="BRILAP4327" failures="3" tests="6" name="Test" time="0.358" errors="0" timestamp="2022-03-02T12:59:11 IST">
  <testcase classname="azureCICD.Test1" name="failme" time="0.012"/>
  <testcase classname="azureCICD.Test1" name="printme" time="0.002"/>
  <testcase classname="azureCICD.Test1" name="whoisme" time="0.001"/>
  <testcase classname="azureCICD.Test2" name="testcaseimplement2" time="0.191">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at azureCICD.Test2.testcaseimplement2(Test2.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testcaseimplement2 -->
  <testcase classname="azureCICD.Test3" name="testcaseimplement3" time="0.041">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at azureCICD.Test3.testcaseimplement3(Test3.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testcaseimplement3 -->
  <testcase classname="azureCICD.Test4" name="ReaddataAndCompare" time="0.034">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at azureCICD.Test4.ReaddataAndCompare(Test4.java:76)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ReaddataAndCompare -->
  <testcase classname="azureCICD.Test1" name="skipme" time="0.0">
    <ignored/>
  </testcase> <!-- skipme -->
</testsuite> <!-- Test -->
